pkg/bindings/test/transformer.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/stretchr/testify/require"
```

Comment:
IMO This file should continue to live in `"github.com/cloudevents/sdk-go/pkg/binding/test"` (`RunTransformerTestArgs` and `RunTransformerTests`)